src/components/ReanimatedDrawerLayout.tsx

```diff
@@ -0,0 +1,756 @@
+            (event.translationX +
+              (drawerOpened ? drawerWidth * -gestureOrientation : 0));
+
+          dragX.value =
+            sideCorrection *
+            Math.max(
+              drawerOpened ? startedOutsideTranslation : 0,
+              startedInsideTranslation
+            );
+
+          drawerTranslation.value =
+            drawerType === DrawerType.FRONT
+              ? sideCorrectedDragX.value +
+                interpolate(
+                  -1 * sideCorrectedDragX.value,
+                  [drawerWidth - 1, drawerWidth, drawerWidth + 1],
+                  [0, 0, 1]
+                )
+              : sideCorrectedDragX.value;
+        })
+        .onEnd(handleRelease);
+    }, [
+      dragX,
+      drawerLockMode,
+      drawerTranslation,
+      drawerType,
+      drawerWidth,
+      emitStateChanged,
+      gestureOrientation,
+      handleRelease,
+      edgeHitSlop,
+      fillHitSlop,
+      minSwipeDistance,
+      props.hideStatusBar,
+      props.keyboardDismissMode,
+      sideCorrectedDragX.value,
+      overlayDismissGesture,
+      drawerOpened,
+      isFromLeft,
+      containerWidth,
+      sideCorrection,
+      drawerState,
+      activeCursor,
+      enableTrackpadTwoFingerGesture,
+      mouseButton,
+      statusBarAnimation,
+    ]);
+
+    // When using RTL, row and row-reverse flex directions are flipped.
+    const reverseContentDirection = I18nManager.isRTL
+      ? isFromLeft
+      : !isFromLeft;
+
+    const dynamicDrawerStyles = {
+      backgroundColor: drawerBackgroundColor,
+      width: drawerWidth,
+    };
+
+    const containerStyles = useAnimatedStyle(() => {
+      if (drawerType === DrawerType.FRONT) {
+        return {};
+      }
+
+      return {
+        transform: [
+          {
+            translateX: interpolate(
+              openValue.value,
+              [0, 1],
+              [0, drawerWidth * sideCorrection],
+              Extrapolation.CLAMP
+            ),
+          },
+        ],
+      };
+    });
+
+    const drawerAnimatedStyle = useAnimatedStyle(() => {
+      const closedDrawerOffset = drawerWidth * -sideCorrection;
+      const isBack = drawerType === DrawerType.BACK;
+      const isIdle = drawerState === DrawerState.IDLE;
+
+      if (isBack) {
+        return {
+          transform: [
+            {
+              translateX: 0,
+            },
+          ],
+          flexDirection: reverseContentDirection ? 'row-reverse' : 'row',
+        };
+      }
+
+      let translateX = 0;
+
+      if (isIdle) {
+        translateX = drawerOpened ? 0 : closedDrawerOffset;
+      } else {
+        translateX = interpolate(
+          openValue.value,
+          [0, 1],
+          [closedDrawerOffset, 0],
+          Extrapolation.CLAMP
+        );
+      }
+
+      return {
+        transform: [
+          {
+            translateX,
+          },
+        ],
+        flexDirection: reverseContentDirection ? 'row-reverse' : 'row',
+      };
+    });
```

Comment:
I know it's just formatting, but can we stick to this:

```tsx
{
  transform: [{ translateX: 0 }],
  flexDirection: reverseContentDirection ? 'row-reverse' : 'row',
};
```

so it is 4 lines long instead of 9? 😅

Reply:
Changed in [3e36cbe](https://github.com/software-mansion/react-native-gesture-handler/pull/3146/commits/3e36cbe15fb8f37ec262ef7268026e43cad70eaf).
I had no idea allowed making any style changes without adding a `// prettier-ignore` comment 😅